operations/exports.md

```diff
@@ -0,0 +1,158 @@
+# Exports
+
+> ### Restricted!
+> These operations are currently in beta-test and as such are subject to change.
+
+## Get all exports
+
+Get exports by provided `ExportIds`. Note this operation supports [Portfolio Access Tokens](../guidelines/multi-property.md).
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/exports/getAll`
+
+```json
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "ExportIds": [
+        "3fa85f64-5717-4562-b3fd-2c963f66afa6",
+        "09708665-0e31-4b23-b337-b0a000be0df0",
+        "706dc6d5-9511-4751-825e-538ce99da2ce",
+        "f776f20a-6f1a-4ddf-93f4-9dae95261415"
+    ]
```

Comment:
I think you are missing `Limitation` in the example? It's required

Reply:
No, per the last discussion it was removed since don't allow listing on this endpoint. See also CON-648.

Reply:
If that so, remove it from the list of properties below, where it say required 👍